WiinUPro/Windows/MainWindow.xaml.cs

```diff
@@ -375,12 +375,17 @@ private void settingSuppressLostConn_Checked(object sender, RoutedEventArgs e)
         private void settingAutoAddXInputDevices_Checked(object sender, RoutedEventArgs e)
         {
             AppPrefs.Instance.autoAddXInputDevices = settingAutoAddXInputDevices.IsChecked ?? false;
+            settingProfileQueuing.IsEnabled = settingAutoAddXInputDevices.IsChecked ?? false;

```

Comment:
The option of Profile Queueing is now only enabled when autoAddXInputDevices is, since the Profile Queueing will not work particularly well without it.

Reply:
That makes sense. Although I think the way this is setup is that if you check or uncheck the auto add XInput devices then it also checks or unchecks auto profile queuing. So if profile queuing is off and then auto add XInput gets checked, profile queuing automatically gets turned on which probably shouldn't be the expected behvaior.

Here I think the only other operation we want to do is check if auto add xinput is being turned off and if it is make sure profile queuing also gets turned off.

Reply:
Oh my bad, this is right. you're changing IsEnabled, all good!